src/components/ImportFromCognitoS3Dialog/set-url.js

```diff
@@ -0,0 +1,43 @@
+import RecognizeFileExtension from "../../utils/RecognizeFileExtension"
+const setUrl = (result, configImport) => {
+  if (configImport.loadProjectIsSelected) {
+    if (RecognizeFileExtension(result) === "Image") {
```

Comment:
RecognizeFileExtension doesn't follow a regular naming convention. You should use something like

`getDataTypeOfUrl`

Or better yet, use the mime-types module so you're not reinventing the wheel! https://www.npmjs.com/package/mime-types